Server/src/main/resources/static/js/editor.js

```diff
@@ -175,6 +175,53 @@ function parseDWInstruction(line, result, currentLine) {
     }
 }
 
+function isLabel(text, result) {
+    for (i = 0; i < result.labels.length; i++) {
```

Comment:
use `var i` instead of implicitly declaring it

Reply:
Any reason to not use `let`? Since `var` kinda sucks with `for` and `if` blocks.

Reply:
For consistency, since it's var Everywhere else in the file.